### PR TITLE
Added kube state metrics config in prometheus.

### PR DESCRIPTION
**Pull request by Arnav15**
Add ability to query kube_state_metrics in Kubernetes clusters.

### CODE REVIEW

File: stable/yugaware/templates/configs.yaml [hunk 1]
```diff
@@ -191,6 +191,24 @@ data:
             target_label: "saved_name"
             replacement: "$1"
 
+      - job_name: 'kube-state-metrics'
+        static_configs:
+        - targets: ['kube-state-metrics.kube-system.svc.cluster.local:8080']
```
Comment (ramkumarvs) [inline stable/yugaware/templates/configs.yaml]: what if the cluster domain name is different?

File: stable/yugaware/templates/configs.yaml [hunk 2]
```diff
@@ -191,6 +191,24 @@ data:
             target_label: "saved_name"
             replacement: "$1"
 
+      - job_name: 'kube-state-metrics'
```
Comment (ramkumarvs) [inline stable/yugaware/templates/configs.yaml:194]: do we need any changes to our RBAC permissions to query kube state metrics?

**Review by ramkumarvs — CHANGES_REQUESTED** [review]
can you also bump the chart version and image tag?

File: stable/yugaware/templates/configs.yaml (hunk 1 above)
Comment (Arnav15) [reply stable/yugaware/templates/configs.yaml]: Addressed comment.

File: stable/yugaware/templates/configs.yaml (hunk 2 above)
Comment (Arnav15) [reply stable/yugaware/templates/configs.yaml:194]: No, that can be setup when bringing up the kube_state_metrics service.

**Review by ramkumarvs — APPROVED** [review]

**Pull request merged**